lib/active_admin/filters/humanized.rb

```diff
@@ -10,7 +10,7 @@ def initialize(param)
       end
 
       def value
-        @value
+        @value.is_a?(::Array) ? @value.compact.join(', ') : @value
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.